coderd/database/dbfake/databasefake.go

```diff
@@ -933,7 +933,7 @@ func (q *fakeQuerier) GetUsers(_ context.Context, params database.GetUsersParams
 
 	// Database orders by username
 	slices.SortFunc(users, func(a, b database.User) bool {
-		return a.Username < b.Username
+		return strings.ToLower(a.Username) < strings.ToLower(b.Username)
```

Comment:
I hope not, but can we have multiple users named the same? `ZeroCool`, `zerocool`, `Zerocool` and `zeroCool`. So that we'd need to handle `lower(a) == lower(b)` case?

Reply:
Fortunately: `User already exists.` :)